content/blog/2024-05-21-Audit.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "Security Audit for Eclipse Kuksa released"
+date = "2024-05-21"
+tags = ["KUKSA.val", "Audit", "Security"]
+categories = ["Implementation"]
+banner = "img/banners/2024-05-21-auditBanner.png"
++++
+
+Applications that utilize artifacts from the Kuksa project need to rely on the secure execution of these components. We are, therefore, happy to announce the publication of an external audit report carried out by [Quarkslab](https://www.quarkslab.com). The audit was facilitated through [OSTIF](https://ostif.org) and the [Eclipse Foundation](https://www.eclipse.org) and made possible by the founding the Eclipse Foundation received from the [Alpha-Omega project](https://openssf.org/community/alpha-omega/).
+
+- The audit was scoped on the KUKSA databroker and the respective Python client SDK.
+- There were 19 findings with low or higher severity, of which two had a high severity.
+- The KUKSA team fixed most findings before the release of the report.
+
+In this post, we explain some of the findings and how they were addressed. The full audit report is [available here](https://ostif.org/wp-content/uploads/2024/05/Kuksaaudit1.2.pdf)
+
+## Audit process
+
+Together with the auditors, there was an agreement to focus the analysis on the KUKSA databroker and the Python client SDK since these are the most used deliverables of the Eclipse Kuksa project. The team at Quarkslab then based their work on the threat model and extended this through automated and manual static analysis. In addition, they performed dynamic fuzz testing. After completing the report, the auditors shared their results with the Kuksa team. The maintainers of Eclipse Kuksa then resolved all findings with low or higher severity without another review by Quarkslab. We can now share some examples here for findings and their fixes.
+
+## Findings
+
+There were two findings for which the auditors assigned a high severity:
```

Comment:
Maybe also give one sentence to the low findings. Maybe give some examples on what those findings were

Reply:
My idea was to mention this below under "Further findings"